archlinux/PKGBUILD.install

```diff
@@ -64,89 +62,89 @@ configure_notification-daemon() {
 ## Post-Install functions ##
 ############################
 
 update_qubesconfig() {
+    # Create NetworkManager configuration if we do not have it
+    if ! [ -e /etc/NetworkManager/NetworkManager.conf ]; then
+    echo '[main]' > /etc/NetworkManager/NetworkManager.conf
+    echo 'plugins = keyfile' >> /etc/NetworkManager/NetworkManager.conf
+    echo '[keyfile]' >> /etc/NetworkManager/NetworkManager.conf
+    fi
+    /usr/lib/qubes/qubes-fix-nm-conf.sh
+
+    # Remove ip_forward setting from sysctl, so NM will not reset it
+    # Archlinux now use sysctl.d/ instead of sysctl.conf
+    #sed 's/^net.ipv4.ip_forward.*/#\0/'  -i /etc/sysctl.conf
 
-	# Create NetworkManager configuration if we do not have it
-	if ! [ -e /etc/NetworkManager/NetworkManager.conf ]; then
-	echo '[main]' > /etc/NetworkManager/NetworkManager.conf
-	echo 'plugins = keyfile' >> /etc/NetworkManager/NetworkManager.conf
-	echo '[keyfile]' >> /etc/NetworkManager/NetworkManager.conf
-	fi
-	/usr/lib/qubes/qubes-fix-nm-conf.sh
-
-	# Remove ip_forward setting from sysctl, so NM will not reset it
-	# Archlinux now use sysctl.d/ instead of sysctl.conf
-	#sed 's/^net.ipv4.ip_forward.*/#\0/'  -i /etc/sysctl.conf
-
-	# Remove old firmware updates link
-	if [ -L /lib/firmware/updates ]; then
-	  rm -f /lib/firmware/updates
-	fi
-
-	# Yum proxy configuration is fedora specific
-	#if ! grep -q '/etc/yum\.conf\.d/qubes-proxy\.conf' /etc/yum.conf; then
-	#  echo >> /etc/yum.conf
-	#  echo '# Yum does not support inclusion of config dir...' >> /etc/yum.conf
-	#  echo 'include=file:///etc/yum.conf.d/qubes-proxy.conf' >> /etc/yum.conf
-	#fi
-	#/usr/lib/qubes/update-proxy-configs
-	# Archlinux pacman configuration is handled in update_finalize
-
-	# Location of files which contains list of protected files
-	mkdir -p /etc/qubes/protected-files.d
-	. /usr/lib/qubes/init/functions
-
-	# qubes-core-vm has been broken for some time - it overrides /etc/hosts; restore original content
-	if ! is_protected_file /etc/hosts ; then
-	    if ! grep -q localhost /etc/hosts; then
-	      cat <<EOF > /etc/hosts
-	127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4 `hostname`
-	::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
+    # Remove old firmware updates link
+    if [ -L /lib/firmware/updates ]; then
+      rm -f /lib/firmware/updates
+    fi
+
+    # Yum proxy configuration is fedora specific
+    #if ! grep -q '/etc/yum\.conf\.d/qubes-proxy\.conf' /etc/yum.conf; then
+    #  echo >> /etc/yum.conf
+    #  echo '# Yum does not support inclusion of config dir...' >> /etc/yum.conf
+    #  echo 'include=file:///etc/yum.conf.d/qubes-proxy.conf' >> /etc/yum.conf
+    #fi
+    #/usr/lib/qubes/update-proxy-configs
+    # Archlinux pacman configuration is handled in update_finalize
+
+    # Location of files which contains list of protected files
+    mkdir -p /etc/qubes/protected-files.d
+    # shellcheck source=../init/functions
```

Comment:
shellcheck is called from source top dir and it expect paths relative from there...